tests/functionaltests/Tests/NSBundleTests.mm

```diff
@@ -19,39 +19,54 @@
 #import <UWP/WindowsFoundation.h>
 #import <UWP/WindowsStorage.h>
 
-TEST(NSBundle, MSAppxURL) {
-    LOG_INFO("MSAppxURL test: ");
-
-    NSURL* resourceURL = [[NSBundle mainBundle] URLForResource:@"FunctionalTests" withExtension:@"dll"];
-    NSURL* msAppxURL = [[NSBundle mainBundle] _msAppxURLForResourceWithURL:resourceURL];
-    ASSERT_OBJCEQ([NSURL URLWithString:[@"ms-appx:///" stringByAppendingString:[resourceURL lastPathComponent]]], msAppxURL);
-
-    WFUri* msAppxURI = [WFUri makeUri:msAppxURL.absoluteString];
-
-    // Verify that the file can be accessed
-    __block BOOL success = NO;
-    __block BOOL signaled = NO;
-    __block NSCondition* condition = [[NSCondition new] autorelease];
-
-    [WSStorageFile getFileFromApplicationUriAsync:msAppxURI
-        success:^void(WSStorageFile* file) {
-            [condition lock];
-            success = YES;
-            signaled = YES;
-            [condition signal];
-            [condition unlock];
-        }
-        failure:^void(NSError* error) {
-            LOG_ERROR([[error description] UTF8String]);
-            [condition lock];
-            signaled = YES;
-            [condition signal];
-            [condition unlock];
-        }];
-
-    [condition lock];
-    ASSERT_TRUE (signaled || [condition waitUntilDate:[NSDate dateWithTimeIntervalSinceNow:2]]);
-    [condition unlock];
-
-    ASSERT_EQ(YES, success);
-}
\ No newline at end of file
+class NSBundleTests {
+public:
+    BEGIN_TEST_CLASS(NSBundleTests)
+    END_TEST_CLASS()
```

Comment:
just moved over from functionaltests.cpp but yes that would be cleaner.

---
In reply to: [92248343](https://github.com/Microsoft/WinObjC/pull/1523#discussion_r92248343) [](ancestors = 92248343,92248298)